pkg/products/cloudresources/reconciler.go

```diff
@@ -504,3 +534,75 @@ func (r *Reconciler) reconcileCIDRValue(ctx context.Context, client k8sclient.Cl
 
 	return client.Patch(ctx, cfgMap, k8sclient.Merge)
 }
+
+// passArnIntoSecretInCroNamespace pass the ARN value into a Secret in CRO namespace
+func passArnIntoSecretInCroNamespace(ctx context.Context, client k8sclient.Client, operatorNamespace string, stsRoleArn string) error {
+	// create CRO credentials secret
+	credSec := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      croSecretName,
+			Namespace: operatorNamespace,
+		},
+		Data: map[string][]byte{},
+	}
+
+	_, err := controllerutil.CreateOrUpdate(context.TODO(), client, credSec, func() error {
+		credSec.Data["AWS_ROLE_ARN"] = []byte(stsRoleArn)
```

Comment:
```suggestion
		credSec.Data["role_arn"] = []byte(stsRoleArn)
		credSec.Data["web_identity_token_file"] = []byte("/var/run/secrets/openshift/serviceaccount/token")
```
Can you implement this^ change, please?
That^ is how CRO expects it at this point.
I should have communicated secret format upfront, sorry about that.

Reply:
Done. Thank you for comment.